CoordImporter/Windows/MainWindow.cs

```diff
@@ -24,7 +24,7 @@ public MainWindow(HuntHelperManager huntHelperManager) : base("Coordinate Import
     {
         this.SizeConstraints = new WindowSizeConstraints
```

Comment:
In this file let's just pull over the manky code I made on top of the PR ( https://github.com/zw3lf/ffxiv-coord-importer/commit/c8e9ceb4190f7e9d07155ef9b0a6d8006e58e8fc ) you originally had to address the scaling problems. IIRC the combined work we did got the scaling issue resolved and it made it look OK